documentation/config_logging_advanced.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: "Puppet Server: Advanced Logging Configuration"
+canonical: "/puppetserver/latest/config_logging_advanced.html"
+---
+
+Puppet Server uses the very flexible, very powerful [Logback](http://logback.qos.ch/) library under the hood to handle all of our logging needs.
```

Comment:
Added, though, since this is only on the `master` branch, I don't think I necessarily need to add the old path?  So I only added the `/etc/puppetlabs/puppetserver` one.